index.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module "fhirpath" {
+  export function evaluate(
```

Comment:
parse and compile are missing.
See exported members here https://github.com/HL7/fhirpath.js/blob/master/src/fhirpath.js#L693.

BTW don't define type for ucumUtils, it is not the part of library.